gmaven-runtime/pom.xml

```diff
@@ -40,6 +40,10 @@
         <module>gmaven-runtime-api</module>
         <module>gmaven-runtime-support</module>
         <module>gmaven-runtime-1.5</module>
+        <module>gmaven-runtime-1.6</module>
+        <module>gmaven-runtime-1.7</module>
+        <module>gmaven-runtime-1.8</module>
+        <module>gmaven-runtime-2.0</module>
```

Comment:
profiles turn these on, any reason why these were added here?

Reply:
No, I just didn't realize there were profiles (and plain 'mvn install' complained about missing snapshot of gmaven-runtime-1.6). I removed that change - thanks.

Reply:
maybe the profiles aren't kicking on properly as they should, will have a look when I merge this. thx.

Reply:
I guess it's because the 1.x branch is old and unaware of jdk's >1.7. Since I built it with Java 8, no profile was matched to the JDK. Maybe I should have added a section for java8, but when I simply used -Pjava7 everything built fine.